test/contexts/SignedInUserContext.test.jsx

```diff
@@ -59,12 +58,12 @@ describe('SignedInUserContext', () => {
     });
     fetchProfileInfo.mockResolvedValue({ profileInfo: {} });
     getPodUrlAll.mockResolvedValue(['https://example.com/pod/']);
-    const { container } = render(
+    const { findByText } = render(
       <SignedInUserContextProvider>
         <TestConsumer />
       </SignedInUserContextProvider>
     );
-    await flushPromises();
-    expect(container).toMatchSnapshot();
+    const val = await findByText('https://example.com/pod/');
+    expect(val).not.toBeNull();
```

Comment:
We were only using the snapshot to check if the podUrl was correctly rendered. We may as well just check it directly. Since we're not using the snapshot anymore, I deleted it.